deployment/cfn/application.py

```diff
@@ -17,14 +17,11 @@
 from utils.constants import (
     ALLOW_ALL_CIDR,
     EC2_INSTANCE_TYPES,
-    GRAPHITE,
```

Comment:
All of the changes in this file should really not be in this PR because they are staging specific concerns. I ended up removing them because I was interactively going through find/replace.

Actually testing that these changes are working should be deferred to another PR creation/review cycle.